From 0202970e6027964a0277b24350a3fe93c9bdb214 Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Thu, 4 Jun 2015 16:34:28 +0800
Subject: [PATCH] MLK-11030-2 ARM: dts: change SAI MCLK to 36864000HZ to
 support 32bit 32k format for SAI master mode

Change SAI MCLK to 36864000HZ to support 32bit 32k format for SAI master
mode.

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 49781eb..7e8a1a5 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -942,7 +942,7 @@
 	assigned-clocks = <&clks IMX7D_SAI1_ROOT_SRC>,
 			  <&clks IMX7D_SAI1_ROOT_CLK>;
 	assigned-clock-parents = <&clks IMX7D_PLL_AUDIO_POST_DIV>;
-	assigned-clock-rates = <0>, <18432000>;
+	assigned-clock-rates = <0>, <36864000>;
 	status = "okay";
 };
 
-- 
1.8.0

